### PR TITLE
XIVY-6828 Split meta.json and product.json

### PR DESCRIPTION
**Pull request by alexsuter**

### CODE REVIEW

File: market/connector/currencyConverter/product.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "properties": {
+        "openApiUrl": "openapi.json"
+    },
+    "installers": [
+        {
+            "id": "rest-client",
+            "data": {
+                "openApiUrl": "@openApiUrl",
```
Comment (ivy-rew) [inline market/connector/currencyConverter/product.json:9]: I think with the version we have a slightly different than our ${version} ... if we sould have an end separator it's easier to parse I think

**Review by ivy-rew — APPROVED** [review]
let's think about the properties expression pattern again

File: market/connector/microsoft365/calendar/product.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            "...": "<../msgraph/project-res.json>"
+        },
+        {
+            "...": "<../msgraph/rest-client.json>"
+        },
+        {
+            "id": "project-resource",
+            "data": {
+                "dialog": {
+                    "source": "@main@demo/msGraphDemo/src_hd",
```
Comment (ivy-rew) [inline market/connector/microsoft365/calendar/product.json:19]: ahh I see ... I already inventend that other pattern :-(   :dancers:

Comment (ivy-rew) [reply market/connector/microsoft365/calendar/product.json:19]: then it loooks to me as if the designer is already aware of that featureset ... and we can keep it as is. sorry for the confusion

**Review by ivy-rew — APPROVED** [review]

File: market/connector/microsoft365/calendar/product.json (hunk 2 above)
Comment (alexsuter) [reply market/connector/microsoft365/calendar/product.json:19]: ahh but the pattern is with closing tag I think

**Pull request merged**